crates/sui-node/src/lib.rs

```diff
@@ -365,6 +366,18 @@ impl SuiNode {
             .is_transaction_executed_in_checkpoint(digest)
     }
 
+    pub fn get_transaction_checkpoint(
```

Comment:
i think this method probably belongs in AuthorityState

Reply:
yes and let's share code with https://github.com/MystenLabs/sui/pull/8619 maybe